core-services/prow/02_config/_plugins.yaml

```diff
@@ -5058,6 +5079,34 @@ plugins:
     - verify-owners
     - wip
     - yuks
+  konveyor/mig-operator:
+    plugins:
+    - assign
+    - blunderbuss
+    - bugzilla
```

Comment:
I don't think we want the bugzilla plugin

Reply:
There are bunch of plugins here, that are really usefull for the merge bot, but I don't think that we ae using the merge bot right? Things like jira, lgtm/hold/override/wip/approve  I don't think are needed.

Reply:
+1 you are on the mark! I will remove them